Sources/ComposableArchitecture/Reducer/Reducers/StackReducer.swift

```diff
@@ -707,11 +714,13 @@ public struct _StackReducer<Base: Reducer, Destination: Reducer>: Reducer {
 /// Notice that after removing all elements and appending a new element, the ID generated was 2 and
 /// did not go back to 0. This is because in tests the IDs are _generational_, which means they
 /// keep counting up, even if you remove elements from the stack.
-public struct StackElementID: Hashable, Sendable {
+public struct StackElementID: Hashable, @unchecked Sendable {
```

Comment:
It is possible to mandate that `elementIdentifier` conform to `Sendable`, while `NavigationLink.value` only requires `Hashable`.
Further consideration is warranted in this matter.